5.machine_learning/0.train_logistic_regression/split_and_train.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# initialize the correct shell for your machine to allow conda to work (see README for note on shell names)
+conda init bash
+# activate the machine learning conda environment
+conda activate machine_learning_cfret
+
+# Go into the first module for training models
+cd 0.train_logisitic_regression
+
+# convert all notebooks to python files into the scripts folder
```

Comment:
FWIW, I recommend naming this folder `nbconverted` or `nbconverted_scripts` so that someone knows exactly what to expect in this folder (probably a future change for future projects). A `scripts` folder might mean something else (e.g., like a `utils` folder) to someone else.

Reply:
Ooo I haven't thought of that. I agree that I think this is something I can easily implement in future projects or ones that are in the beginning stages (e.g., ALSF and GFF). I might consider making a PR prior to a new release that goes through to make that change. I will make this an issue!